src/systems/userinput/userinput.js

```diff
@@ -22,36 +21,113 @@ import { viveUserBindings } from "./bindings/vive-user";
 import { xboxControllerUserBindings } from "./bindings/xbox-controller-user";
 import { daydreamUserBindings } from "./bindings/daydream-user";
 
-import { updateActionSetsBasedOnSuperhands } from "./resolve-action-sets";
+import { resolveActionSets } from "./resolve-action-sets";
 import { GamepadDevice } from "./devices/gamepad";
 import { gamepadBindings } from "./bindings/generic-gamepad";
 
-const priorityMap = new Map();
-function prioritizeBindings(registeredMappings, activeSets) {
-  const activeBindings = new Set();
-  priorityMap.clear();
-  for (const mapping of registeredMappings) {
+const satisfiesPath = (binding, path) => {
+  return Object.values(binding.dest) && Object.values(binding.dest).indexOf(path) !== -1;
```

Comment:
Maybe a little suspicious performance here.

Reply:
The `indexOf` can't be replaced with `===` because we want a binding with `/foo/bar` to "satisfy" any path like `foo/bar/baz`. Dom pointed out that `indexOf` isn't right because we actually want to only check whether the two paths start with the same words between the `/`s... 

e.g. "/foo/ba" should not "satisfy" "/foo/bar" but right now it does

Reply:
Oh, I see. I forgot about that part.